pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/test/java/org/apache/pinot/plugin/minion/tasks/segmentgenerationandpush/SegmentGenerationAndPushTaskGeneratorTest.java

```diff
@@ -78,4 +86,52 @@ public void testRealCluster()
     ControllerTest.sendPostRequest(_controllerRequestURLBuilder.forClusterConfigs(), request);
     Assert.assertEquals(_generator.getNumConcurrentTasksPerInstance(), 1);
   }
+
+  @Test
+  public void testGenerateTaskSpec() throws Exception {
+    URL resourcesLoc = SegmentGenerationAndPushTaskGeneratorTest.class.getClassLoader().getResource(".");
+    SegmentGenerationAndPushTaskExecutor executor = new SegmentGenerationAndPushTaskExecutor();
+    Schema schema = new Schema.SchemaBuilder().build();
+    // FieldUtils.writeField(executor, "_pinotTaskConfig", new PinotTaskConfig("",
```

Comment:
(nit) Can we remove this line?

Reply:
Fixed in bb90840fffa9e1f3d2f03af2f8270127bfe91787